mirrord-cli/src/main.rs

```diff
@@ -136,9 +139,10 @@ fn main() {
 
     let cli = Cli::parse();
     match cli.commands {
-        Commands::Exec(args) => exec(&args).unwrap(),
+        Commands::Exec(args) => exec(&args),
         Commands::Extract { path } => {
             extract_library(Some(path));
```

Comment:
I think you would have warning here if you wont add ?

Reply:
Nay - the function returns `Result<()>`, and because we added that to the `main() -> Result<()>` as well, this compiles as is. If I add the `?` operator - `Commands::Exec(args) => exec(&args)?,`, `cargo check` returns the following:

```
error[E0308]: `?` operator has incompatible types
   --> mirrord-cli/src/main.rs:142:33
    |
142 |         Commands::Exec(args) => exec(&args)?,
    |                                 ^^^^^^^^^^^^ expected enum `Result`, found `()`
    |
    = note: `?` operator cannot convert from `()` to `Result<(), anyhow::Error>`
    = note:   expected enum `Result<(), anyhow::Error>`
            found unit type `()`
help: try removing this `?`
    |
142 -         Commands::Exec(args) => exec(&args)?,
142 +         Commands::Exec(args) => exec(&args),
    | 
help: try wrapping the expression in `Ok`
    |
142 |         Commands::Exec(args) => Ok(exec(&args)?),
    |                                 +++            +

For more information about this error, try `rustc --explain E0308`.
error: could not compile `mirrord` due to previous error
```

Reply:
I think it'd be best to have `?` in the end of all arm branches (on exec + Extract) then outside the match branch (in the end of function) just to have `Ok(())` which means that if it got there, we're all good.